backend/plugins/content_api/src/modules/portal/db/models/company.ts

```
@@ -0,0 +1,74 @@
+        ...doc,
+        createdAt: new Date(),
+      });
+    }
+
+    public static async updateCompany(
+      _id: string,
+      doc: IPortalCompanyDocument,
+    ) {
+      await models.Companies.updateOne({ _id }, { $set: doc });
+
+      return this.getCompany(_id);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)